app/(frontend)/(client-side)/layout.tsx

```diff
@@ -1,10 +1,10 @@
 import Navbar from "@/app/components/ui/Navbar";
-import Footer from "@/app/components/ui/footer";
+import Footer from '@/app/components/ui/footer'
 import { ReactNode } from "react";
 
-export default function ClientLayout ({children}: {children: ReactNode}) {
+export default function ClientLayout ({ children }: { children: ReactNode}) {
   return (
-    <div className='flex flex-col min-h-screen justify-between'>
+    <div className="flex flex-col min-h-screen">
```

Comment:
This changes make that in page there is a blank space within the footer and the end of the page

![image](https://github.com/Los-creativos/cloteck-e/assets/90226510/0bd1f78b-260a-4379-9cb6-30b7448750ed)

![image](https://github.com/Los-creativos/cloteck-e/assets/90226510/9c40fe18-b891-49e4-8aa9-522ea902a615)

As is show in the pictures it affects more pages